src/codegen/llvm/Builder.zig

```diff
@@ -2136,6 +2135,122 @@ pub const CallConv = enum(u10) {
     }
 };
 
+pub const StrtabString = enum(u32) {
```

Comment:
MetadataString is still usefull as we need the indices of the metadata strings when emitting metadata, it also makes it much easier to emit the metadata strings as we can just emit the `metadata_string_bytes` array as a blob instead of having to build a blob from accessing normal strings.